src/test_mail_room.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+import pytest
+
+
+STATE4_TABLE = [
+    (10, 10),
+    (11, 11),
+    # ('0', SystemExit),
+]
+# def test_mailroom():
+    # from mailroom_functions import mailroom_functions
+
+
+def test_mailroom_initialize_donor_dict():
+    from mailroom_functions import initialize_donor_dict
+    assert type(initialize_donor_dict()) == dict
+
+
+def test_list_of_donors():
+    from mailroom import list_of_donors
+
+
+def test_state2():
+    from mailroom_functions import state_2_valid_responses
+
+
+def test_state4_1():
+    from mailroom import state_4_valid_responses
+    assert state_4_valid_responses('10') == 10
```

Comment:
If this test feels insufficient to you, it may be because the `state_4_valid_responses` function is operating at the wrong level.  If it doesn't do anything that is testable and interesting, why does it exist?